haproxy18/tests/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+SKIPBUILD=${SKIPBUILD:-0}
+
+hab pkg install core/busybox-static
+hab pkg binlink core/busybox-static netstat
+
+hab pkg install --binlink core/bats
+source "${PLANDIR}/plan.sh"
+
+if [ "${SKIPBUILD}" -eq 0 ]; then
+  set -e
+  pushd "$(dirname "${PLANDIR}")" > /dev/null
+  build "${pkg_name}"
+  source results/last_build.env
+  hab svc load -f core/nginx
+  echo '[http.listen]
+port = 81' | hab config apply nginx.default "$(date +%s)"
+  # Unload the service if its already loaded.
+  hab svc unload "${HAB_ORIGIN}/${pkg_name}"
+  hab pkg install --binlink --force "results/${pkg_artifact}"
+  hab svc load "${pkg_ident}" --bind=backend:nginx.default
+  popd > /dev/null
+  set +e
+fi
+
+echo "Waiting 1 second hoping haproxy will start..."
+sleep 1
```

Comment:
need to increase the sleep on this - maybe 5 seconds or so.

Reply:
changed to 5 secs